hydradx/model/processing.py

```diff
@@ -1,42 +1,85 @@
 import pandas as pd
```

Comment:
We should probably refactor out and add tests for the important parts of the post-processing when time permits (not as a part of this PR)

Reply:
Yes.